### ISSUE TITLE
Portalable Angled Panels aren't styled in Overgrown Style

### ISSUE THREAD

**Issue by cartman1337** [BEEmod/BEE2-items#2460]
This is with 2.4.0pr28. Black (un-portalable) panels look correct.

![Clean angled panels in Overgrown style](https://i.imgur.com/B9BYMHw.jpg)

Is there a quick fix I can apply manually to make the panels look more in style?

**Comment by cartman1337** [comment]
Apparently it applies to some, but not all, Flip Panels too. My latest chamber has several flip panels, and one of them appears clean, while the others appear worn down. All the angled panels appear clean, though.

**Comment by LautaroL20** [comment]
Its weird.

**Comment by cartman1337** [comment]
Done some investigating into this matter. Apparently the problem is the same for both angled and flip panels, but it only applies to panels on the floor or roof, and it is also dependent on if there is a connection to it or not (like from a button, trigger event, etc.)

This image shows a few panels; some with connections from a pedestal button, some without. Those without are styled, those with are clean.
![Panels](https://i.imgur.com/rzj9zuz.png)
There's one exception, though; an angled panel on the floor, in a 0 degree tilt, shows clean and slightly elevated above the ground. Is that (the elevation) intended?

This next image shows a few flip panels and an angled panel on a wall, all with connections:
![More panels](https://i.imgur.com/wBpII1Q.jpg)
The two flip panels on the wall (top of the wall, near the wall) and the angled panel on the wall are all styled, but the flip panels on the ground are clean.

Seems the connection is the main culprit here.

There is, however, one exception. Both flip and angled panels with connections may appear styled if a faith plate points at it:
![Panels with bullseyes](https://i.imgur.com/kOImhpz.jpg)

As a sidenote; why do angled panels with and without connections to them have different mechanisms?
![Different mechanisms on same type of panel](https://i.imgur.com/IEKLHfz.jpg)

**Comment by TeamSpen210** [comment]
The checkered texture is actually Overgrown, it's just less dirty than others.The elevation is intended, it's a bonus I added since otherwise that configuration is pointless. Try a 30 degree one on ceilings or floors.

**Comment by LautaroL20** [comment]
This is intentional then.

**Comment by cartman1337** [comment]
I don't see what's so special about the 30 degree ones.
![30 degree panel](https://i.imgur.com/Np6X5GB.jpg)

As for the checkered texture, I still maintain that it looks out of place, as that texture can't be seen anywhere else in that world. As such I would ask to consider replacing it in a later release anyway.

**Comment by TeamSpen210** [comment]
I coded the panels to behave slightly differently with the angles. Any which never move, at greater than 45° use a wall texture, otherwise they use ceiling/floor textures. For moving panels, currently they use the orientation of the surface they're attached to (ignoring the angle they extend to). I might want to make this texture more dirty.

**Comment by ZombiePwner1** [comment]
Yes they texture could use some dirt on it since it just looks really out of place.
